Jenkinsfile

```diff
@@ -31,17 +31,15 @@ def restoreIngenialinkWheelEnvVar() {
 
 def getWheelPath(tox_skip_install, python_version) {
     if (tox_skip_install) {
-        def stashName = python_version == PYTHON_VERSION_MIN ? "build" : "build_${python_version}"
-        unstash stashName
         script {
-            def distDir = python_version == PYTHON_VERSION_MIN ? "dist" : "dist_${python_version}"
-            def result = bat(script: "dir ${distDir} /b /a-d", returnStdout: true).trim()
-            def files = result.split(/[\r\n]+/)    
-            def wheelFile = files.find { it.endsWith('.whl') }
+            def result = bat(script: "dir dist /b /a-d", returnStdout: true).trim()
+            def files = result.split(/[\r\n]+/)
+            def pythonVersionTag = "cp${python_version.replace('py', '')}"
```

Comment:
Maybe this is easier: https://www.jenkins.io/doc/pipeline/steps/pipeline-utility-steps/#findfiles-find-files-in-the-workspace